.github/workflows/CD.yml

```diff
@@ -595,41 +595,6 @@ jobs:
         if: ${{ steps.release-latest-image.outputs.cache-hit != 'true' && (steps.version.outputs.major-update == 'true' || steps.version.outputs.minor-update == 'true') }}
         run: cd tools/release/adot-operator-images-mirror && go run ./
 
-  clean-ssm-package:
-    runs-on: ubuntu-latest
-    if: ${{ always() }}
-    needs: [ s3-release-validation-1, s3-release-validation-2, s3-release-validation-3, release-checking ]
-    strategy:
```

Comment:
Honestly, we would only need to destroy the strategy. However, if you prefer this way, then maybe delete the [clean-ssm-package need in this line](https://github.com/aws-observability/aws-otel-collector/pull/940/files#diff-a64cbb6e953e697cb4302ab31af9ebbd95a8de6216f874bb0ffdefce8e3c1cd9R600) as well.